woocommerce/woocommerce-gutenberg-products-block
# Testimonials Single Pattern > Update the image format and content width

Update the styles for the image and the content width/padding for the Testimonials Single Pattern as requested by the design team. An alt attribute was also added to provide alternative information for the image if a user for some reason cannot view it



Fixes #10598

### Screenshots



| Before | After |
| ------ | ----- |
| <img width="1202" alt="Screenshot 2023-08-15 at 12 33 09" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/3fd554e1-a527-48d2-9a42-ab17434f0b30"> <img width="333" alt="Screenshot 2023-08-15 at 12 39 30" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/0c5b535b-f248-47b2-89b4-be7af6f3c157"> |  <img width="1200" alt="Screenshot 2023-08-15 at 12 36 31" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/ab4f5fa8-19e5-45a7-a398-fe45bef1b7a6"> <img width="333" alt="Screenshot 2023-08-15 at 12 37 17" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/e2c0a790-3fef-4051-bef3-8fc133f86db1"> |

### Testing

#### User Facing Testing



1. Create a new post
2. Insert the **Testimonials Single** Pattern
3. Make sure the pattern is correctly displayed in the editor
4. Save the post and head over to the front end: make sure the pattern is correctly displayed both on desktop and mobile view (the layout should match the one demonstrated in the above screenshots in the **After** column).

### WooCommerce Visibility




### Performance Impact



### Changelog

> Updated the image format and content width for the Testimonials Single Pattern.
